cmd/gossamer/commands/import_state.go

```diff
@@ -62,7 +64,16 @@ func execImportState(cmd *cobra.Command) error {
 		return fmt.Errorf("header-file must be specified")
 	}
 
+	stateVersionFlag, err := cmd.Flags().GetString("state-version")
+	if err != nil {
+		return fmt.Errorf("failed to get state-version: %s", err)
+	}
+	stateVersion, err := trie.ParseVersion(stateVersionFlag)
+	if err != nil {
+		return fmt.Errorf("failed to parse state-version: %s", err)
```

Comment:
Very good point! Thank you!

Reply:
It's a valid point and I share it but I wanted to follow the convention we have in that file. We are returning `failed` in every error parsing a flag https://github.com/ChainSafe/gossamer/blob/development/cmd/gossamer/commands/import_state.go#L44-L60

Reply:
This is part of the `cmd` package and doesn't print the log with the `ERROR:` prefix like the node does. If we wanna use the same convention here, I believe it'll have to be a separate task.

Reply:
I tend to agree with @kishansagathiya to not include "failed" in the error message. However, since it is already used in this pkg, its my opinion that we leave as is for now, then can change in a separate task if/when we think it is appropriate to.

Reply:
@kishansagathiya are you agree?

Reply:
@dimartiro sorry, I had missed the conversation. This makes sense to me, resolving it.